cmd/minikube/cmd/console.go

```diff
@@ -47,14 +51,26 @@ var consoleCmd = &cobra.Command{
 		}
 		if consoleURLMode {
 			fmt.Fprintln(os.Stdout, url)
+		} else if machineReadAble{
+			hostIP, err := cluster.GetHostIP(api)
+			if err != nil {
+				glog.Errorln("Cannot get Host IP. Verify that Minishift is running. Error: ", err)
+			}
+			displayConsoleInMachineReadable(hostIP, url)
 		} else {
 			fmt.Fprintln(os.Stdout, "Opening the OpenShift Web console in the default browser...")
 			browser.OpenURL(url)
 		}
 	},
 }
 
+func displayConsoleInMachineReadable(hostIP string, url string) {
+	machineDetails = fmt.Sprintf(machineDetails, hostIP, constants.APIServerPort, url)
```

Comment:
I actually found it quite interesting the other day as well, that the port is a constant. I guess there is no big need to change it, but somehow I expected the port to be configurable and then retrievable in some way.